whipper/program/cdparanoia.py

```diff
@@ -196,7 +196,7 @@ def getTrackQuality(self):
 
         # don't go over a 100%; we know cdparanoia reads each frame at least
         # twice
-        return min(frames * 2.0 / reads, 1.0)
+        return min(frames * 2.0 / reads, 1.0) if reads else 0
```

Comment:
Most likely my CD drive was somewhat broken because I didn't even managed to rip one single CD with it but regardless of that; whipper should catch it IMHO.
In the long run one could think about throwing actual error messages if the drive operates weird (like here).